src/extensions/default/Microsoft.DotNet.UpgradeAssistant.Steps.Source/CodeFixerStep.cs

```diff
@@ -155,6 +163,26 @@ protected override async Task<UpgradeStepApplyResult> ApplyImplAsync(IUpgradeCon
             return new UpgradeStepApplyResult(UpgradeStepStatus.Complete, $"No instances of {DiagnosticId} need fixed");
         }
 
+        private void AddResultToContext(IUpgradeContext context, string diagnosticId, string location, string status, string resultMessage)
```

Comment:
is it appropriate for `status` to be `string` or should it follow an enum (perhaps already defined?)

Reply:
Sure if we delegate this to an extension method, an enum would make more sense